extensions-core/s3-extensions/src/main/java/org/apache/druid/storage/s3/S3StorageDruidModule.java

```diff
@@ -195,6 +195,12 @@ public void configure(Binder binder)
       );
     }
 
+    if (storageConfig.getS3TransferConfig() != null) {
+      S3TransferConfig config = storageConfig.getS3TransferConfig();
+      log.debug("s3 transfer config useTransferManager=[%b], minimumUploadPartSize=[%d], multipartUploadThreshold=[%d]",
```

Comment:
You should check, but I believe we already log all the configs on startup, so this log message might not be necessary. Otherwise, I would only log a message if transfer-manager is enabled.

Reply:
Thanks for the feedback! Verified that each config parameter is logged individually on startup. Removed this extra logging statement.